src/niamoto/core/plugins/exporters/index_generator.py

```
@@ -0,0 +1,287 @@
+
+                # Apply filters if configured
+                if config.filters:
+                    include_item = True
+                    for filter_config in config.filters:
+                        filter_value = self._get_nested_value(item, filter_config.field)
+                        if filter_config.operator == "in":
+                            if filter_value not in filter_config.values:
+                                include_item = False
+                                break
+                        elif filter_config.operator == "equals":
+                            if filter_value != filter_config.values[0]:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)